README.md

```diff
@@ -47,121 +47,41 @@ For more information about available options run:
 $ ./_output/bin/descheduler --help
 ```
 
-## Running Descheduler as a Job Inside of a Pod
+## Running Descheduler as a Job or CronJob
 
-Descheduler can be run as a job inside of a pod. It has the advantage of
+The descheduler can be run as a job or cronjob inside of a pod. It has the advantage of
 being able to be run multiple times without needing user intervention.
-Descheduler pod is run as a critical pod to avoid being evicted by itself,
-or by kubelet due to an eviction event. Since critical pods are created in
-`kube-system` namespace, descheduler job and its pod will also be created
+The descheduler pod is run as a critical pod to avoid being evicted by itself,
+or by the kubelet due to an eviction event. Since critical pods are created in the
+`kube-system` namespace, the descheduler job and its pod will also be created
 in `kube-system` namespace.
 
-###  Create a container image
```

Comment:
wonder if we would like to have some instructions for people creating an image from the source?

Reply:
I can re-add this section if you want. Just let me know.

After the v0.10.0 release I was planning doing a series a pull requests to improve the documentation. One thing I wanted to do was create a contributing guide that would cover this. The contributing guide would be put into the `docs` directory. My opinion is the README.md should be optimized for end users, not contributors/developers.

Reply:
makes sense. lets do the way you suggested.